resources/views/filament/pages/dashboard.blade.php

```diff
@@ -70,8 +70,10 @@ class="col-span-full flex flex-col items-center rounded-lg bg-black bg-cover bg-
             @if (UnlicensedNotice::canView())
                 @livewire(UnlicensedNotice::class)
             @else
-                @livewire(ServiceRequestWidget::class)
```

Comment:
Though it was asked in the equivalent Advising App ticket to remove widgets that were on the page, this was not specifically asked in this ticket for Aiding App.

I have asked Product what they want here, please follow up with that and re-request my review once either a change is made or if Product is good with this removal.
https://canyongbs.atlassian.net/browse/AIDAPP-754?focusedCommentId=25124